src/core/scene.ts

```diff
@@ -1136,6 +1137,139 @@ export class Scene {
     });
   }
 
+  /**
+   * return: Promise<string>
+   *
+   * Get the transition override of this scene object.
+   * Transition overrides take priority over the more generic one from App.GetTransition
+   * See also: {@link #core/Transition Core/Transition} and {@link #core/App#getTransition getTransition}
+   *
+   *
+   * #### Usage
+   *
+   * ```javascript
+   * myScene.getTransitionOverride().then(function(transition) {
+   *  // do something here
+   * });
+   * ```
+   */
+  getTransitionOverride(): Promise<Transition> {
+    return new Promise(resolve => {
+      let _sceneId = versionCompare(getVersion()).is.lessThan(sceneUidMinVersion) ? this._id : this._uid;
```

Comment:
I just noticed that almost all other methods had this exact line... couldn't we abstract this into a class property instead?

Reply:
Maybe do it as another issue?
- in order to unclutter changes here
- I think other unmerged changes also use this

Reply:
Yeah, we can do that :)

Reply:
Issue to abstract the above-mentioned line filed as #285. Also, rechecked the branch and unit tests should still be okay. Not really sure why the checks in travis have failed, but I'm confident we can proceed with merging it to 2.10.0 branch